web/packages/teleport/src/services/audit/types.ts

```diff
@@ -317,6 +317,7 @@ export const eventCodes = {
   USER_TASK_CREATE: 'UT001I',
   USER_TASK_UPDATE: 'UT002I',
   USER_TASK_DELETE: 'UT003I',
+  SFTP_SUMMARY: 'TS021I',
```

Comment:
Can you move this to the area where the other SFTP events are declared?

As a side note, I think there are a bunch of unused constants in the code for the SFTP events that we used to emit before the PR landed that removed a bunch of them. We should remove them if we're not going to use them.

Reply:
That's fair I'll do that. I debating putting it with the other SFTP event codes or putting it at the bottom next to new events but either works really